packages/cli/src/databases/migrations/common/1697198787123-AddExecutionHasBinaryData.ts

```diff
@@ -0,0 +1,13 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+const tableName = 'execution_entity';
+
+export class AddExecutionHasBinaryData1697198787123 implements ReversibleMigration {
+	async up({ schemaBuilder: { addColumns, column } }: MigrationContext) {
+		await addColumns(tableName, [column('hasBinaryData').bool.default(false)]);
```

Comment:
The problem I see is that we're not populating existing executions, effectively leaving those files iin the disk forever if using `filesystem` already

Reply:
Definitely! We'd need migrations, or make the skipping exclusive to `s3`.